consensus/state.go

```diff
@@ -1331,19 +1332,14 @@ func (cs *ConsensusState) tryAddVote(vote *types.Vote, peerKey string) error {
 		if err == ErrVoteHeightMismatch {
 			return err
 		} else if _, ok := err.(*types.ErrVoteConflictingVotes); ok {
```

Comment:
this `ok` is about whether or not the error is of a particular type. we use that convention throughout the codebase and should stay consistent